testsuite/tests/index/index.ml

```diff
@@ -0,0 +1,69 @@
+(* TEST
+flags = "-bin-annot -store-usage-index";
+compile_only = "true";
+readonly_files = "auxiliaire.ml";
+setup-ocamlc.byte-build-env;
+all_modules = "auxiliaire.ml index.ml";
+ocamlc.byte;
+check-ocamlc.byte-output;
+program = "-index -decls index.cmt";
+output = "out_objinfo";
+ocamlobjinfo;
+program = "awk '/Indexed/,0' out_objinfo";
```

Comment:
We don't want a direct call to `awk` in the testsuite: nodboy does that (despite this being a dirty hack) so surely this does not work. Our proposal with @Octachron is to extend ocamlobjinfo with a `-quiet` option that silences the printing of everything, except sections that have been explicitly asked for -- and would be used here with `-shapes -index -decls`.)

Reply:
Done in [6f1cd93](https://github.com/ocaml/ocaml/pull/12508/commits/6f1cd93d2267fda8b9e0a271d0390df40966ba1b)